app/categorytree/CategoryTreeProvider.java

```diff
@@ -1,7 +1,7 @@
-package inject;
+package categorytree;
 
 import com.google.inject.Provider;
-import controllers.RefreshableCategoryTree;
+import inject.SunriseInitializationException;
```

Comment:
Do we want to put SunriseInitializationException into another package? Not every initialization problem is related to dependency injection.